### ISSUE TITLE
iOS: Selfhosting settings

### ISSUE THREAD

**Issue by czb** [bitwarden/mobile#236]
Selhosting settings seems broken on iPhone 6 and the latest iOS. See this image:

![image-1](https://user-images.githubusercontent.com/37391/34510519-cd39e738-f054-11e7-80c8-8197db73ba7c.jpg)

**Comment by kspearrin** [comment]
Yes, we're aware of this one and working on a solution at the moment. It is also related to the same problem in #237.
